workers/common/src/main/kotlin/common/env/GitCredentialsGenerator.kt

```diff
@@ -51,15 +51,29 @@ class GitCredentialsGenerator : EnvironmentConfigGenerator<EnvironmentServiceDef
          * the _.git-credentials_ file. Use [builder] to obtain secret references. Return *null* if the URL is invalid.
          */
         private fun InfrastructureService.urlWithCredentials(builder: ConfigFileBuilder): String? = runCatching {
-            buildString {
-                val serviceUrl = URI.create(url).toURL()
+            val serviceUrl = URI.create(url).toURL()
+
+            val githubCredentials = buildString {
```

Comment:
Nit: The variable name is misleading. This is not related to GitHub, but a plain Git configuration file. Maybe just `entry`.